rero_ils/modules/fees/api.py

```diff
@@ -118,8 +118,8 @@ def create_fee_from_notification(cls, notification):
                 holding_circulation_category_pid
             )
             data['amount'] = cipo.get('overdue_amount')
-            data['currency'] = current_app.config\
-                .get('RERO_ILS_DEFAULT_CURRENCY')
+            data['currency'] = notification.organisation.\
```

Comment:
Should we have a property `default_currency` in the `notification`class?

Reply:
Aly confirm to me than in rero_ils, never a loan has no linked item (-->locations-->library-->organisation). But, as loan could be empty from the `loan` jsonschema, I added again a `default_currency` into global `config.py`.